src/Components/Map.js

```diff
@@ -41,7 +41,7 @@ class Map extends Component{
         this.map = new mapboxgl.Map({
         container: this.mapContainer,
         style: 'mapbox://styles/mapbox/streets-v11',
-        center: [21.43141,41.99646],
+        center: [55.8596,-4.2503],
```

Comment:
I take it this is changing the location to Glasgow as well?

Reply:
Ah yes, apologies. Forgot I'd put that in. Zoom level isn't sorted yet.